packages/elements-core/src/components/TryIt/TryIt.spec.tsx

```diff
@@ -240,10 +240,11 @@ describe('TryIt', () => {
       // path param
       const completedField = screen.getByLabelText('completed');
       expect(completedField).toHaveValue('');
+      expect(completedField).toHaveTextContent('select an option');
 
       // query params
       const limitField = screen.getByLabelText('limit');
-      expect(limitField).toHaveTextContent('select an option');
+      expect(limitField).toHaveTextContent('defaults to: 1');
```

Comment:
i think we also discussed `select an option (defaults to: 1)`

Reply:
i thought about it myself, but i didnt have such information in the ticket, good change imo